hw/lint/tools/ascentlint/parse-lint-report.py

```diff
@@ -32,38 +32,75 @@ def get_results(resdir):
     """
     results = {
         "tool": "ascentlint",
+        "fusesoc-error": [],
```

Comment:
this last commit basically replicates what @rswarbrick has done in the first commit for the the Verible style lint parser